kb/2022-03-10/multiple_nics_vm_connectivity.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Multiple NICs VM Connectivity
+descripion: How to deal VMs with multiple NICs in Harvester
+slug: multiple-nics-vm-connectivity
+authors:
+  - name: Date Huang
+    title: Software Engineer
+    url: https://github.com/tjjh89017
+    image_url: https://github.com/tjjh89017.png
+tags: [vm, network]
+hide_table_of_contents: false
+---
+
+# Multiple NICs VM Connectivity
+
+## What is the default behavior of a VM with multiple NICs
+
+In some scenarios, you'll setup two or more NICs in your VM to serve different networking purposes. If all networks are setup by default with DHCP, you might get random connectivity issues. And while it might get fixed after rebooting the VM, it still will lose connection randomly after some period.
+
+## How-to identify connectivity issues
+
+In a Linux VM, you can use commands from the `iproute2` package to identify the default route.
+
+In your VM, execute the following command:
+```bash
+ip route show default
+```
+!!! note
+    If you get the `access denied` error, please run the command using `sudo`
+    
+The command's output will display only the `default` route with the gateway and VM IP of the main network interface (`eth0` in the example below).
+```
+default via <Gateway IP> dev eth0 proto dhcp src <VM IP> metric 100
+```
+
+Here is the full example:
+```
+$ ip route show default
+default via 192.168.0.254 dev eth0 proto dhcp src 192.168.0.100 metric 100
+$
+```
```

Comment:
I thought we could just show the full example here because this is the basic stuff.

Reply:
but it will depend on different setup.
not easy to generate for a full example